backend/module/chaos/serverexperimentation/rtds/integration/Dockerfile-envoy

```
@@ -0,0 +1,13 @@
+from golang:1.15.6 as build
```

nit: i guess compose is permissive but none of the syntax highlighters or linters seem to like lowercase directives
```suggestion
FROM golang:1.15.6 as build
```